### PR TITLE
react-week1

### PR DESCRIPTION
**Pull request by mahmutkaya**

### CODE REVIEW

File: documentation/todo-app/src/App.js [hunk 1]
```diff
@@ -1,18 +1,14 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
 import './App.css';
+import TodoList from './components/todoList';
```
Comment (NouranMahmoud) [inline documentation/todo-app/src/App.js]: It's better to start the name of the component with Capital letter `import TodoList from './components/TodoList';`

File: documentation/todo-app/src/components/todoItems.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React, { Component } from "react";
+
+class TodoItems extends Component {
+    render() {
+        const description = this.props.description;
+        const deadline = this.props.deadline;
```
Comment (NouranMahmoud) [inline documentation/todo-app/src/components/todoItems.js]: You can use destructuring `const {description, deadline} = this.props;`

File: documentation/todo-app/src/components/todoList.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+                'deadline': 'Fri Sep 15 2017'
+            }
+        ];
+
+        const todoItems = todoS.map((task, index) => (
+            <TodoItems description={task.description} deadline={task.deadline} />
+        ));
+        return (
+            <div className='todoList'>
+                <tr>
```
Comment (NouranMahmoud) [inline documentation/todo-app/src/components/todoList.js]: Why are you using `<tr>` however you are not using `<table>`. Anyway table is not a good idea :)

Comment (mahmutkaya) [reply documentation/todo-app/src/components/todoList.js]: Thank you for feedback Nouran, I have fixed some parts, and I will do the rest soon :)

**Pull request closed without merging**